pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{lib, stdenv, fetchFromGitHub, bash, fakeroot, libinput, xdotool, makeWrapper}:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  buildInputs = [ bash fakeroot makeWrapper ];
+
+  installPhase =
+    ''
+      mkdir -p $out
+      fakeroot -- bash libinput-gestures-setup -d "$out" install
+      mv $out/usr/bin $out/bin-unwrapped
```

Comment:
When you call ```makeWrapper```/```wrapProgram``` later, it will take care of renaming instead you having to define your own scheme.

Reply:
I used that at first, but the config files are discovered depending on argv[0]. Using wrapProgram means it will look for `{/etc,~/.config}/.libinput-gestures-wrapped.conf`, rather than `{/etc,~/.config}/libinput-gestures.conf`.

Reply:
```wrapProgram``` takes an ```--argv SOME_NAME``` argument that should deal with that.

Reply:
Looks like `wrapProgram` already sets `--argv0`. I tried setting it manually, but that still didn't work.